monetio/models/gocart.py

```diff
@@ -0,0 +1,75 @@
+"""GOCART File Reader."""
+
+import xarray as xr
+
+
+def open_mfdataset(fnames):
+    """Method to open GOCART netcdf files.
+
+    Parameters
+    ----------
+    fnames
+        String glob expression or a list of files to open.
+
+    Returns
+    -------
+    xarray.Dataset
+        GOCART model dataset in standard format for use in MELODIES MONET
+
+    """
+
+    ds = xr.open_mfdataset(
+        fnames,
+        concat_dim="time",
+        combine="nested",
+        drop_variables=["AOD_BC", "AOD_DU", "AOD_OC", "AOD_SS", "AOD_SU"],
```

Comment:
Sure! I hadn't thought of a good naming convention since these variables are also wavelength-dependent - for example could do "AOD550_DU" or something else?

Reply:
Maybe just always stick the wavelength on the end of the initial name?